lkmpg.tex

```diff
@@ -190,7 +190,9 @@ \subsection{Before We Begin}
 
         If you got this message, the simplest way is to disable the UEFI SecureBoot from the PC/laptop boot menu to have your "hello-1" to be inserted.
         Of course you can go through complicated steps to generate keys, install keys to your system, and finally sign your module to make it work.
-        However, this needs another chapter for the advanced users but not suitable for beginners.
+        However, this is not suitable for beginners.
+        Please follow the steps in \href{https://ubuntu.com/blog/how-to-sign-things-for-secure-boot}{how-to-sign-things-for-secure-boot} if you are interested.
```

Comment:
Don't change the title of a specified source. That is, you shall remain "How to sign things for Secure Boot" for `\href`.